src/react-vite/11-controlled-vs-uncontrolled/controlled-vs-uncontrolled.md

```diff
@@ -2,22 +2,20 @@
 @parent learn-react-vite 11
 @outline 3
 
-@description TODO
+@description Use forms and inputs to collect information.
 
 @body
 
 ## Overview
 
-- Passing data back from a component to its parent using a functional prop
-- TODO
-
-## Objective 1
+## Objective 1: Add a form with controlled checkboxes
 
 TODO
 
 ### Key concepts
 
-TODO
+- A controlled input requires value (checked) and onChange props.
+- Store form data as an object in state.
```

Comment:
This has been changed to be setting state using the `setState(callback function)` signature.